scroll-animations-1/Overview.bs

```diff
@@ -542,25 +508,9 @@ as follows:
 1.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
     in the direction specified by {{orientation}}.
 
-1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
-    time value if {{fill}} is <code>none</code> or <code>forwards</code>, or 0 otherwise.
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return -Infinity.
```

Comment:
So if I understand correctly the output is not fully decided yet. I expect this to change once the discussion is resolved here:
https://github.com/w3c/csswg-drafts/issues/2066#issuecomment-580384210